docs/template.sql

```diff
@@ -0,0 +1,28 @@
+-- ===============================================================
+-- Description:
+-- Here you describe what the example does.
+-- Your description should follow the following format:
+-- In this query, <description>.
+
+-- If the example has multiple steps, then number them as follows,
+-- In this query, <description>:
+-- 1. <step one>
+-- 2. <step two>
+-- ...
+-- ===============================================================
+
+-- Variable creation:
+-- Describe what variables are being defined.
+-- For this project, we use the `luminesce-examples` scope.
+
+-- Example:
+-- Defining portfolio scope and code.
+
+@@portfolioScope = select 'luminesce-examples';
+@@portfolioCode = select 'uk-equity';
+
+-- Example of querying instruments
+
+select * from Lusid.Portfolio 
+where PortfolioCode=@@portfolioCode
+and PortfolioScope=@@portfolioScope;
```

Comment:
It would be good to have an example of writing to LUSID through a provider too. I seem to remember this being more fleshed out and not referencing real providers? Correct me if I am wrong though, been a week or so...

Reply:
Stephen requested to limit it all down to this for barebones template

Reply:
I think you need to mark this issue as resolved, I can't seem to find the option to do it on my end. Approved otherwise